README.md

```diff
@@ -142,6 +142,10 @@ Currently we have:
 + ClinVar benigns are either `Benign` or `Likely-Benign` and variants with uncertainty are removed.
 + ClinVar variants where there is an SSR field are removed because they are suspected false positives due to paralogy or computational/sequencing error
 
+#### Benchmarks
```

Comment:
why is this called `Benchmarks`? Isn't it just another clinvar truth set?

Reply:
please remove this new sub-section from the README and consolidate the below paragraph into a single sentence.